content/docs/user-guide/scripting/script-canvas/programmer-guide/custom-nodes/custom-nodeable-nodes.md

```diff
@@ -0,0 +1,259 @@
+---
+linktitle: Custom Nodeable Nodes
+title: Creating Custom Nodeable Nodes in Script Canvas
+description: Learn how to create custom Script Canvas Nodeable Nodes in Open 3D Engine (O3DE).
+weight: 100
+---
+
+This topic guides you how to create custom Script Canvas Nodeable Nodes step by step.
+
+You'll see the term _nodeable_ used throughout the O3DE source. 
+A nodeable can refer to both the node that appears in the Node Palette as a result of the AzAutoGen processing,
+and the mechanism by which a compiled Script Canvas graph can invoke C++ functions.
+
+## Prerequisite: Adding support for custom nodeable nodes to a Gem
+In your Gem's `CMakeLists.txt`, add a section for `AUTOGEN_RULES` and declare `Gem::ScriptCanvas` as build dependency.
+
+The precise place for this section will vary depending on how your Gem is configured. 
+However, we recommend that your Gem define a `STATIC` library to make the code available to both runtime and editor projects.
+
+As an example, here is partial definition of Gem's `CMakeLists.txt` that supports Script Canvas custom nodes with following required changes:
+1. `Gem::ScriptCanvas` must be declared as `BUILD_DEPENDENCIES` of `STATIC` library
+2. Add `AUTOGEN_RULES` section for custom free function under `STATIC` library
+   ```cmake
+   AUTOGEN_RULES
+       *.ScriptCanvasNodeable.xml,ScriptCanvasNodeable_Header.jinja,$path/$fileprefix.generated.h
+       *.ScriptCanvasNodeable.xml,ScriptCanvasNodeable_Source.jinja,$path/$fileprefix.generated.cpp
+   ```
+3. `STATIC` library must be declared directly/indirectly as `BUILD_DEPENDENCIES` of Gem runtime/editor module 
```

Comment:
Instead of using slashes ("/") in "directly/indirectly" and "runtime/editor", be explicit about what you mean. Slashes are not always clear to a reader.

Reply:
done